src/test/java/POM/HomePage.java

```diff
@@ -0,0 +1,26 @@
+package POM;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class HomePage extends BasePage {
+
+//    WebDriver driver;
+//    WebDriverWait wait;
+    By userAvatarIcon = By.cssSelector("img.avatar");
+
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+//        driver = givenDriver;
+//        wait = new WebDriverWait(driver, Duration.ofSeconds(5));
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.

![image](https://user-images.githubusercontent.com/104761482/207938212-b500dfc5-a4ff-4873-b42c-cbf454a0e87f.png)